packages/core/upload/admin/src/components/EditFolderDialog/EditFolderDialog.js

```diff
@@ -0,0 +1,215 @@
+import * as yup from 'yup';
+import { Formik } from 'formik';
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import isEmpty from 'lodash/isEmpty';
+import { useIntl } from 'react-intl';
+import { Button } from '@strapi/design-system/Button';
+import { Grid, GridItem } from '@strapi/design-system/Grid';
+import {
+  ModalLayout,
+  ModalHeader,
+  ModalBody,
+  ModalFooter,
+} from '@strapi/design-system/ModalLayout';
+import { FieldLabel } from '@strapi/design-system/Field';
+import { Stack } from '@strapi/design-system/Stack';
+import { TextInput } from '@strapi/design-system/TextInput';
+import { Typography } from '@strapi/design-system/Typography';
+import { VisuallyHidden } from '@strapi/design-system/VisuallyHidden';
+import { Form, useNotification, getAPIInnerErrors } from '@strapi/helper-plugin';
+
+import { getTrad } from '../../utils';
+import { useEditFolder } from '../../hooks/useEditFolder';
+import { ContextInfo } from '../ContextInfo';
+import SelectTree from '../SelectTree';
+
+const folderSchema = yup.object({
+  name: yup.string().required(),
```

Comment:
Don't we need to add some specific validation for the folder name as it will probably parts of urls? 
![Screenshot 2022-05-06 at 17 15 30](https://user-images.githubusercontent.com/7756284/167163978-ecb84958-0c10-4825-8f6f-29c6b07e84da.png)

Reply:
No, the name won't be part of a URL. We store an automatically generated UUID, which is used for the paths. The folder name itself has a lot of validations in the backend, which are passed through.